### ISSUE TITLE
Change pin for CS

### ISSUE THREAD

**Issue by vince87** [petrockblog/ControlBlockService2#31]
hi, is there any way to modify the driver to work in conjunction with vga666 in dpi = 3 then using gpio25 as CS?
Thank you
Vincenzo

**Comment by petrockblog** [comment]
I am afraid, it is not possible. The problem is that the CS signal line is hard wired on the PCB.

**Comment by vince87** [comment]
No, sorry, it's for another project, that use VGA666 mode 3 and a mcp23s17, but the only free pins are the GPIO 9 10 11 25 26 27!
Your driver is perfect, but the CS pin can't be GPIO 7 8!
Thank you
